app/src/main/java/com/it3048/mobiledeviceproject/MainActivity.kt

```diff
@@ -1,38 +1,27 @@
 package com.it3048.mobiledeviceproject
 
-import android.content.DialogInterface
 import android.os.Bundle
-import android.view.ContextThemeWrapper
-import android.view.LayoutInflater
+import android.view.*
```

Comment:
The wildcard operator (*) is not recommended.  
Even though it takes more lines, the preferred method is to import everything explicitly.  1) It's better for documentation.  2) There's less risk of naming clashes.  Example: both java.util.* and java.sql.* have a class called Date.  So, if you import both packages with a wildcard operator, and then reference the class called Date, which one should it use?